### DISCUSSION TITLE
About the license used by mmyolo

### DISCUSSION THREAD

**Discussion by ypatz** [open-mmlab/mmyolo#493] [General]
Hi there,
Most of open-mmlab's repositories use the Apache license.
I'm just wondering why mmyolo uses the GPL license, because of yolo？
Is it possible to use the Apache license？

**Comment by hhaAndroid** [comment]
Because yolov5 v6 v7 v8 are all GPL license, it is impossible to use Apache.

**Reply by ypatz** [reply]
If we not use the code from yolov5 v6 v7 v8 repos, is it possible to use Apache?
Or we have to use GPL as long as we use the yolo v5 v6 v7 v8 structure?

**Reply by hhaAndroid** [reply]
I recommend you to use RTMDet directly, which uses the Apache.
